draft-ietf-protected-audience-key-value-server-api.md

```diff
@@ -1,78 +1,182 @@
 ---
-title: "Protected Audience Key Value Server APIs"
+coding: utf-8
+
+title: "Protected Audience Key Value Service APIs"
 abbrev: "KV Servers APIs"
+docname: draft-ietf-protected-audience-key-value-server-api-latest
 category: std
+submissionType: IETF
 
-docname: draft-ietf-protected-audience-key-value-server-api-latest
-submissiontype: IETF  # also: "independent", "editorial", "IAB", or "IRTF"
-number:
-date:
-consensus: true
-v: 3
-# area: AREA
-# workgroup: WG Working Group
-keyword:
- - protected audience
- - fledge
-venue:
-#  group: WG
-#  type: Working Group
-#  mail: WG@example.com
-#  arch: https://example.com/WG
-  github: "privacysandbox/draft-ietf-protected-audience-key-value-service"
-  latest: "https://privacysandbox.github.io/draft-ietf-protected-audience-key-value-service/draft-ietf-protected-audience-key-value-server-api.html"
+area: TBD
+workgroup: TBD
+keyword: Internet-Draft
+
+ipr: trust200902
+
+stand_alone: yes
+pi: [toc, sortrefs, symrefs]
 
 author:
  -
-    fullname: "Peiwen Hu"
+    name: Peiwen Hu
     organization: Google
-    email: "peiwenhu@google.com"
+    email: peiwenhu@google.com
  -
-    fullname: "Benjamin Russ Hamilton"
+    ins: B. R. Hamilton
+    name: Benjamin "Russ" Hamilton
     organization: Google
-    email:  "behamilton@google.com"
+    email: behamilton@google.com
 
 normative:
-  CBOR: RFC8949
-  CDDL: RFC8610
+  RFC8949: CBOR
+  RFC8610: CDDL
+  RFC8259: JSON
+  RFC9458: OHTTP
+  RFC9180: HPKE
+  RFC1952: GZIP
+  RFC7932: Brotli
 
 informative:
```

Comment:
Why are all of these under "informative" references? Informative references are used in explanations of use cases and applications. They are not supposed to describe how to implement parts of a standard. The ones listed below also seem essential to describing the standard, so they should be normative references. Perhaps just replace the current "normative" references with these references and leave "informative" empty.

Reply:
Done.